doc/api/process.md

```diff
@@ -2516,8 +2516,7 @@ if (process.getuid) {
 }
 ```
 
-This function is only available on POSIX platforms (i.e. not Windows or
-Android).
+This function not available on Windows.
```

Comment:
Is this change related to this PR? Could this land separately, and should we update the YAML history section?

Reply:
It was already like that but docs were not updated so I did it here.